.travis.yml

```diff
@@ -21,27 +21,30 @@ env:
   global:
   - SNAPCRAFT_ENABLE_ERROR_REPORTING=0
 
+before_install:
+# determine build properties and targets
+- |
+  if [ -n "${TRAVIS_PULL_REQUEST}" ]; then
+    export MULTIPASS_BUILD_ID="pr${TRAVIS_PULL_REQUEST}"
```

Comment:
I can make it BUILD_TAG or something. Didn't want to make it too specific, cause it could very well be "ricab" in your local builds :)

That said, they are identifiers as per [the semver spec](https://semver.org/#spec-item-9) :)

Reply:
OK, yeah that sounds good.

Reply:
Which one, then? `MULTIPASS_BUILD_TAG`?

Reply:
Yes, or perhaps MULTIPASS_BUILD_LABEL if you want to prevent people relating it to git tags.

Reply:
Done.